chrobalt/docker/linux/Dockerfile

```diff
@@ -29,20 +31,20 @@ RUN apt-get update && apt-get install -y \
 
 ADD files/sccache /usr/local/bin
 
-# === Install pinned host Clang toolchain for all Linux-hosted builds
-ARG CLANG_VER
+# === Install Clang toolchain for all Linux-hosted builds
+ARG STARBOARD_CLANG_VER
 
 ARG TC_ROOT=${HOME}/starboard-toolchains/
-ARG TC_HOME=${TC_ROOT}/x86_64-linux-gnu-clang-chromium-${CLANG_VER}
-ARG BASE_URL=https://commondatastorage.googleapis.com/chromium-browser-clang
+ARG STARBOARD_CLANG_TC_HOME=${TC_ROOT}/x86_64-linux-gnu-clang-chromium-${STARBOARD_CLANG_VER}
+ARG STARBOARD_CLANG_BASE_URL=https://commondatastorage.googleapis.com/chromium-browser-clang
```

Comment:
Wait - i'm actually confused. Isn't this installing another copy of Chromium toolchain, which Chromium build already uses ?

_starboard_ toolchain is expected to be the target or system toolchain, not the one downloaded, no?